asc.jl

```diff
@@ -131,7 +123,7 @@ S_bcs = FieldBoundaryConditions(top = salt_flux_bc)
 ##### Coriolis
 #####
 
-const f₀ = -1e-4     # [s⁻¹]
+const f₀ = -1.31e-4     # [s⁻¹]
 const β =  1e-11    # [m⁻¹ s⁻¹]
 coriolis = BetaPlane(; f₀, β)
```

Comment:
Seems like Stewart and Thompson have an f-plane at 64S.
We could have that, or a beta-plane at by calling `coriolis = FPlane(latitude = -64)` or `coriolis = BetaPlane(latitude = -64)`. Therefore, 

```suggestion
coriolis = BetaPlane(latitude = -64)
```

Reply:
```Julia
julia> coriolis = BetaPlane(latitude = -64)
BetaPlane{Float64}(f₀=-0.000131082, β=1.0035e-11)
```

Reply:
You can commit my suggestion by pressing the "Commit suggestion" button above ;)